frontend/src/app/my/mentorship/programs/[programKey]/modules/create/page.tsx

```diff
@@ -94,7 +94,32 @@ const CreateModulePage = () => {
         mentorLogins: parseCommaSeparated(formData.mentorLogins),
       }
 
-      await createModule({ variables: { input } })
+      await createModule({
+        variables: { input },
+        update: (cache, { data: mutationData }) => {
```

Comment:
For create logic we still need to have custom update module because as stated [here](https://www.apollographql.com/docs/react/data/mutations#include-modified-objects-in-mutation-responses:~:text=Returning%20modified%20objects,update%20function.) auto-update would not work with any new objects in a list (for example, list of modules in a program)